From c36a6a7f5060e46fc1c6353e31a5a4abaed93be4 Mon Sep 17 00:00:00 2001
From: Pascal Huerst <pascal.huerst@gmail.com>
Date: Thu, 16 Apr 2015 18:17:27 +0200
Subject: [PATCH] fw_env: Adapted for the raumfeld setup

fw_env is the tool to write the environment from userspace.
This adapts the tool to out partition scheme.
---
 tools/env/fw_env.h | 22 +++++++++++-----------
 1 file changed, 11 insertions(+), 11 deletions(-)

diff --git a/tools/env/fw_env.h b/tools/env/fw_env.h
index aff471b..1174d67 100644
--- a/tools/env/fw_env.h
+++ b/tools/env/fw_env.h
@@ -20,20 +20,20 @@
  * See included "fw_env.config" sample file
  * for notes on configuration.
  */
-#define CONFIG_FILE     "/etc/fw_env.config"
+// #define CONFIG_FILE     "/etc/fw_env.config"

 #ifndef CONFIG_FILE
-#define HAVE_REDUND /* For systems with 2 env sectors */
-#define DEVICE1_NAME      "/dev/mtd1"
-#define DEVICE2_NAME      "/dev/mtd2"
-#define DEVICE1_OFFSET    0x0000
-#define ENV1_SIZE         0x4000
-#define DEVICE1_ESIZE     0x4000
+// #define HAVE_REDUND /* For systems with 2 env sectors */
+#define DEVICE1_NAME      "/dev/mtd5"
+// #define DEVICE2_NAME      "/dev/mtd2"
+#define DEVICE1_OFFSET    0x0
+#define ENV1_SIZE         0x20000
+#define DEVICE1_ESIZE     0x20000
 #define DEVICE1_ENVSECTORS     2
-#define DEVICE2_OFFSET    0x0000
-#define ENV2_SIZE         0x4000
-#define DEVICE2_ESIZE     0x4000
-#define DEVICE2_ENVSECTORS     2
+// #define DEVICE2_OFFSET    0x0000
+// #define ENV2_SIZE         0x4000
+// #define DEVICE2_ESIZE     0x4000
+// #define DEVICE2_ENVSECTORS     2
 #endif

 #ifndef CONFIG_BAUDRATE
--
1.9.3

